packages/reports/addon/templates/components/filter-values/range-input.hbs

```diff
@@ -1,24 +1,26 @@
-{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
 {{#if @isCollapsed}}
   {{#if @filter.validations.attrs.values.isInvalid}}
     <FilterValues::InvalidValue/>
   {{else}}
     {{concat @filter.values.firstObject " and " @filter.values.lastObject}}
   {{/if}}
 {{else}}
-  <Input
-    @value={{readonly @filter.values.firstObject}}
-    @placeholder={{this.lowPlaceholder}}
-    class="filter-values--range-input__input {{if @filter.validations.attrs.values.isInvalid "filter-values--range-input__input--error"}}"
-    @key-up={{action "setLowValue"}}
-  />
+  <div class="filter-values--range-input">
+    <Input
+      @value={{@filter.values.firstObject}}
```

Comment:
Did readonly cause an issue?

Reply:
The `input` event makes it only one-way binding instead of two so I figured it's not necessary.